packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -13,6 +13,7 @@ const WebSocket = require('ws')
 const program = require('commander')
 const validator = require('validator')
 const {execSync: _execSync} = require('child_process')
+const projectPkg = require(process.cwd() + '/package.json')
```

Comment:
This caused a regression for MRT log tailing which doesn't necessarily execute from a working directory context with a `package.json` guaranteed to exist